Jordan_Class.py

```diff
@@ -0,0 +1,202 @@
+        self.values_corona = list(map(lambda x:x[1],library_corona))
+        self.keys_meditating = list(map(lambda x:re.compile(x[0], re.IGNORECASE), library_meditating))
+        self.values_meditating = list(map(lambda x:x[1],library_meditating))
+        
+    def smalltalk_Jordan(self):
+        """
+        
+        A function for chitchatting with Jordan.
+        
+        """
+        while True:
+            userInput = input(">>> ") # allows for user to type in something
+            resp = "I did not understand what you said. Also, I am sensitive to punctuation." # initalize response string in case Jordan does not recognize the words
+            counter = 0  # initialize counter 
+            while resp == "I did not understand what you said. Also, I am sensitive to punctuation." and counter < len(self.keys_smalltalk):
+                for i in range(0, len(self.keys_smalltalk)):  # loop through the indexed library
+                    match = self.keys_smalltalk[i].match(userInput) # look if input of the user matches with one of the words in the library
+                    if match: 
+                        word = self.keys_smalltalk[i].split(userInput)[1]  # We have to take the first element of this list                        
+                        resp = random.choice(self.values_smalltalk[i]) # if there is a match, pick a random answer from x[1]
+                    counter += 1 # make sure that the counter is now + 1 so it does not write the initialized response from the beginning but continues with the loop.
+                                 # if there is no match though, then it will write the initialized answer
+            if userInput == 'q':
+                print(random.choice(self.values_smalltalk[i]))
+                print("---------------------------------")
+                print("Do you want to choose another topic? Pick below or press 'q' to quit for realsies.")
+                print("---------------------------------")
+                break                                
+            resp = resp.format(word) # replace if applicable      
+            print(resp) # print response
+                
+    def caring_Jordan(self):
+        """
+        
+        A function to talk with Jordan about your deepest sorrows
+        and your destroyed soul.
+        
+        """
+        
+        while True:
+            userInput = input(">>>")
+            resp = "I did not understand what you said. Also, I am sensitive to punctuation." # initalize this
+            counter = 0  # initalize this
+            while resp == "I did not understand what you said. Also, I am sensitive to punctuation." and counter < len(self.keys_caring):
+                for i in range(0, len(self.keys_caring)):
+                    match = self.keys_caring[i].match(userInput)
+                    if match:
+                        word = self.keys_caring[i].split(userInput)[1]  # We have to take the first element of this list
```

Comment:
see above 

(If this is actually correct code, e.g., because the first element is a placeholder, rewrite your comment. The fewer things the teachers will be confused by, the better the grade, I'd presume)